src/lib.rs

```diff
@@ -35,7 +35,7 @@ pub trait CurveProjective:
 
     /// Normalizes a slice of projective elements so that
     /// conversion to affine is cheap.
-    fn batch_normalization(v: &mut [Self]);
+    fn batch_normalization<S: ::std::borrow::BorrowMut<Self>>(v: &mut [S]);
```

Comment:
This change would now apply to `Curve::batch_normalize`, and would look something like:
```rust
use core::borrow::{Borrow, BorrowMut};
...
fn batch_normalize<P: Borrow<Self>, Q: BorrowMut<Self::AffineRepr>>(p: &[P], q: &mut [Q]) {
```
However, when I tried this locally, I ran into issues with `&P` not implementing `Curve`, that I wasn't immediately able to figure out how to resolve.